velox/common/memory/ByteStream.h

```diff
@@ -383,6 +383,15 @@ inline Timestamp ByteStream::read<Timestamp>() {
   return value;
 }
 
+template <>
+inline int128_t ByteStream::read<int128_t>() {
+  int128_t value = 0;
+  for (int32_t i = 0; i < sizeof(int128_t) / sizeof(uint64_t); ++i) {
```

Comment:
I think this loop is overkill and also hard to read. 128 will always be twice 64. So we can just say
```
value |= static_cast<int128_t>(read<uint64_t>() << 0);
value |= static_cast<int128_t>(read<uint64_t>() << 64);
```